.codespell-ignore

```diff
@@ -0,0 +1,13 @@
+afterall
+casesenstive
```

Comment:
Isn't this actually misspelled? It should be case sens**i**tive

Reply:
Oh! Yes! Thanks! I will remove it.